pkg/process/io.go

```diff
@@ -34,14 +34,17 @@ import (
 	"github.com/containerd/containerd/log"
 	"github.com/containerd/containerd/namespaces"
 	"github.com/containerd/containerd/pkg/stdio"
+	"github.com/containerd/containerd/pkg/timeout"
 	"github.com/containerd/containerd/sys"
 	"github.com/containerd/fifo"
 	runc "github.com/containerd/go-runc"
 	"github.com/hashicorp/go-multierror"
 	"github.com/pkg/errors"
 )
 
-const binaryIOProcTermTimeout = 12 * time.Second // Give logger process solid 10 seconds for cleanup
```

Comment:
if 12 seconds is to long for the default maybe just go back to the original 1sec, at least until we resolve the test buckets using binary IO and expecting a smaller timeout.